### DISCUSSION TITLE
ease of looting mobs when they die on top of each other?

### DISCUSSION THREAD

**Discussion by divinarius** [chromiecraft/chromiecraft#3273] [Suggestions]
My suggestion if possible is to have mobs that are up to 15y between them when they die to be added to a single loot window and see all the loot it can help in raids and dungeons by not missing some loot because of the mob diversity in size and death animations, i know this is a thing that came later in wow but i my honest believe CC can make a wotlk that would even leave blizzard speechless.

**Comment by gettheboom** [comment]
I personally see this as a great idea and making this server stand out from the others.
